tests/crypto/psa_crypto/src/aead.c

```diff
@@ -61,16 +66,22 @@ ZTEST(psa_crypto_test, test_aead_aes_ccm)
 	psa_set_key_usage_flags(&attributes, PSA_KEY_USAGE_ENCRYPT | PSA_KEY_USAGE_DECRYPT);
 	psa_set_key_algorithm(&attributes, alg);
 
+	#if defined(CONFIG_TEST_WRAPPED_KEYS) && CONFIG_TEST_WRAPPED_KEYS
+		printf("Test Wrapper keys enabled\n");
+		psa_set_key_lifetime(&attributes, PSA_KEY_LIFETIME_FROM_PERSISTENCE_AND_LOCATION(
+							  PSA_KEY_PERSISTENCE_VOLATILE, PSA_KEY_VOLATILE_PERSISTENT_WRAP_IMPORT));
```

Comment:
Where does `PSA_KEY_VOLATILE_PERSISTENT_WRAP_IMPORT` come from? This does not seem to be a standard PSA key location. How does this work on Series 2?